authors/apps/articles/models.py

```diff
@@ -47,6 +47,16 @@ def save(self, *args, **kwargs):
                                 uuid.uuid4().hex[:6])
         super().save(*args, **kwargs)
 
+    @property
+    def get_time_to_read(self):
+        # Set the standard read time
+        words_per_min = 270
+        # Cleaning the post content and lower casing all words
+        for char in '-.,\n':
```

Comment:
@pngg please give us descriptive comments about what is going on in this piece of code

Reply:
I agree. I am thinking of implementing a condition to return `Less than a min` if time is less than 1.